.evergreen/evergreen.yml.in

```diff
@@ -1230,6 +1232,28 @@ tasks:
           executable_os_id: <% out(executableOsId) %>
       - func: run_perf_tests
   <% } %>
+  <% for (const arch of ['arm64', 'x64']) {
+      for (const openssl3 of ['1', '0']) {
+    %>
+  - name: e2e_tests_ubuntu2404_<% out(arch) %><% out(openssl3 === '1' ? '_openssl3' : '') %>
+    tags: ["e2e-test"]
+    depends_on:
+      - name: compile_artifact
+        variant: linux_<% out(arch) %>_build<% out(openssl3 === '1' ? '_openssl3' : '') %>
+    commands:
+      - func: checkout
+      - func: install
+        vars:
+          node_js_version: "<% out(NODE_JS_VERSION_20) %>"
+      - func: download_compiled_artifact
+        vars:
+          executable_os_id: linux-<% out(arch) %><% out(openssl3 === '1' ? '-openssl3' : '') %>
+      - func: run_e2e_tests
+        vars:
+          node_js_version: "<% out(NODE_JS_VERSION_20) %>"
+          mongosh_server_test_version: "<% out(SERVER_80_RC_VERSION) %>-enterprise"
+          disable_openssl_shared_config_for_bundled_openssl: ${disable_openssl_shared_config_for_bundled_openssl|false}
+  <% } } %>
```

Comment:
Could this be an addition to `for (const mVersion of ['stable', '6.0.x']) {` in L1197? Or is there something special about Ubuntu 24.04 specifically?

Reply:
thanks. there's nothing special, adding it there created more tasks. after inspecting 6.0, i realized we have unused tasks from that as well and its fine to do it! (i had thought we are using all of them).